### PR TITLE
Added Contributing guideline template

### PR DESCRIPTION
**Pull request by bijoy26**
I have cloned a template from **JU-ForkMe** and adjusted it according to the repo's needs and linked it on the **README.md**.
Check it out.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Open **Git Bash** (or some other terminal) and type the following git command:
+
+```
+git clone "URL"
+```
+Replace *URL* with the copied link.It should look like the following -
+```
+git clone "https://github.com/Arnab-Shanta-Anu/download-all-repos.git"
+```
+You will see a new folder named *JU-ForkMe* has been downloaded to your machine on the **directory** that *Git Bash* is on.
```
Comment (Arnab-Shanta-Anu) [inline CONTRIBUTING.md]: plz correct the repo name *JU-ForkMe to download-all-repos

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Step-3 : Local Changes
+From your machine, your may play with the code and update the necessary changes to the files. <br>
+
+## Step-4 : Commit & Push To Github
+Next you need to push your local repository to your github repository. Write these commands on your terminal:
+
+    ```
+    git add -A
+    git commit -m "your name"
```
Comment (Arnab-Shanta-Anu) [inline CONTRIBUTING.md]: this should say username

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<br>See if your github repository is on the the *head-repository* and it is pointing to the *base-repository* (which is *Arnab-Shanta-Anu/download-all-repos*).
+<br>Give a pull request **title** and **description**<br>
+
+Then click on **Create Pull Request**. <br>
+
+### You will be promted that Pull request has been successfully created.
+
+## Step-7 : Ping & Connect
+
+Wait patiently till the PR is reviewed! <br>
```
Comment (Arnab-Shanta-Anu) [inline CONTRIBUTING.md]: add contact info

File: README.md [hunk 4]
```diff
@@ -39,6 +39,12 @@ follow on screen instructions
 For windows users:
 just double click the exe file
 
+## Contribute to Download-All-Repo
+Suggest your changes by following the [Contributing Guideline](CONTRIBUTING.md).
+
+## Contributors
+Testing, beautification & contributing guideline - [bijoy26](https://github.com/bijoy26)
```
Comment (Arnab-Shanta-Anu) [inline README.md:46]: liking it more

File: README.md [hunk 5]
```diff
@@ -47,6 +53,6 @@ just double click the exe file
 <img src="https://img.shields.io/badge/Facebook:-Arnob%20Shanto%20Anu-blue" alt="Facebook:arnob.shanto.anu" />
 </a>
 
-
-## Contribution
-[bijoy](https://github.com/bijoy26) - beautification & testing
+## Credits
+- Badge sheilds - [shields.io](https://shields.io/)
+- Contributing guideline template collected from [JU-ForkMe](https://github.com/JU-Computer-Club/JU-ForkMe).
```
Comment (Arnab-Shanta-Anu) [inline README.md:58]: nice addition forgot about it

**Review by Arnab-Shanta-Anu — COMMENTED** [review]
plz do the commented changes

File: CONTRIBUTING.md (hunk 1 above)
Comment (bijoy26) [reply CONTRIBUTING.md]: Corrected.

File: CONTRIBUTING.md (hunk 2 above)
Comment (bijoy26) [reply CONTRIBUTING.md]: Corrected.

File: CONTRIBUTING.md (hunk 3 above)
Comment (bijoy26) [reply CONTRIBUTING.md]: Updated.

**Comment by bijoy26** [comment]
@Arnab-Shanta-Anu check the latest revision.

**Review by Arnab-Shanta-Anu — APPROVED** [review]
everything is fine and dandy 😉

**Pull request merged**